klant/src/main/kotlin/com/ritense/portal/klant/autoconfiguration/KlantAutoConfiguration.kt

```diff
@@ -41,7 +41,7 @@ class KlantAutoConfiguration {
     }
 
     @Bean
-    @ConditionalOnMissingBean(OpenKlantClientProvider::class)
+    @ConditionalOnMissingBean(OpenKlantClient::class)
```

Comment:
waarom moet dit nu opeens conditional zijn op OpenKlantClient, is altijd OpenKlantClientProvider geweest en dat lijkt mij ook het meest logisch aangezien je niets met de OpenKlantClient Bean heeft als de OpenKlantClientProvider Bean er niet is

Reply:
@JanBrek jij hebt hier naar gekeken toch?